paths.js

```diff
@@ -1,3 +1,7 @@
+const _ = require('lodash');
+
```

Comment:
Can we use the destructuring assignment please: `const { get } = require('lodash');`

Reply:
Happy to do that, but what is exactly the rationale behind it? Just curious.

Reply:
Readability and consistency throughout Reform.